src/frontend-v2/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Stage 1 - Build Angular application
+FROM node:alpine as build
```

Comment:
Try to use the patterns followed here if you can (including the Google base image): https://github.com/GoogleCloudPlatform/nodejs-docker

We should try to follow the Google container security guidelines when possible

Reply:
Cleaned up the Docker file, added the security guidelines, and used Google base nginx image. I'm having issues getting the node dependencies to install using the nodejs Google base image though, which is preventing the app from compiling successfully so I kept node:alpine for now. Any ideas?